### DISCUSSION TITLE
Number field doesn't accept 0 or negative values

### DISCUSSION THREAD

**Discussion by Jinksi** [community/community#78082] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

To reproduce, create a new `Number` custom field on a GH Project (same occurs for an existing field).

Open an issue on the board. Try to modify the field input's value to `0` or `-10`. Neither will work with this UI. 

**Screen recording:**

https://github.com/community/community/assets/3147296/13b76b85-81f1-4692-80d8-7aa6760f18dc

**Field settings:**

<img width="390" alt="image" src="https://github.com/community/community/assets/3147296/bb3e9235-7bab-459a-9918-e9fe00682aae">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by azenMatt** [comment]
Thanks for reporting this - we are looking into it at the moment

**Reply by Jinksi** [reply]
Thanks @azenMatt 👏

**Comment by azenMatt** [comment, marked as answer]
This is now fixed! Thanks for spotting it 😄

**Comment by Jinksi** [comment]
@azenMatt It seems this issue has returned.

https://github.com/user-attachments/assets/d5d94549-e7eb-4ad1-9069-f4eefe587b03

**Reply by evi-liu** [reply]
Thank you for reporting! Our team is looking into this.

**Reply by evi-liu** [reply]
@Jinksi The fix for this is now merged 🚀! Thanks again for reporting, and let us know if you run into any other bugs.

**Reply by Jinksi** [reply]
Great, thanks for resolving so quickly!
